File: src/main/java/eterea/core/api/rest/repository/IConceptoFacturadoRepository.java
/**
 * 
 */
package eterea.core.api.rest.repository;

import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

import eterea.core.api.rest.model.ConceptoFacturado;

/**
 * @author daniel
 *
 */
@Repository
public interface IConceptoFacturadoRepository extends JpaRepository<ConceptoFacturado, Long> {

}

File: src/main/java/eterea/core/api/rest/kotlin/repository/IConceptoFacturadoRepository.kt
package eterea.core.api.rest.kotlin.repository

import eterea.core.api.rest.kotlin.model.ConceptoFacturado
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository
import java.util.Optional

@Repository
interface IConceptoFacturadoRepository : JpaRepository<ConceptoFacturado?, Long?> {

    fun findByArticuloMovimientoId(articuloMovimientoId: Long): Optional<ConceptoFacturado?>?

}
